### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kidonng**
- ~~Fix languages of some code blocks~~
- Add a missing colon
- Fix some typos in the Deno 1.8 blog post

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -260,7 +260,7 @@ const InstallSection = () => {
     <div key="powershell" className="my-4 text-gray-700">
       <p className="mb-2">PowerShell (Windows):</p>
       <CodeBlock
-        language="bash"
+        language="powershell"
```
Comment (kt3k) [inline pages/index.tsx:263]: This doesn't look improving the rendering of these code blocks, rather it seems introducing inconsistency of the `$` sign before each line. I'd prefer to keep these as is.

**Comment by kt3k** [comment]
The colon fix and typo fixes look good to me.

File: pages/index.tsx (hunk 1 above)
Comment (kidonng) [reply pages/index.tsx:263]: I didn't notice this, makes sense!

**Review by ry — APPROVED** [review]
Thanks

**Pull request merged**